Week1/homework/js2-exercises/4-theTime.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Klok :)</title>
+    <style>
+        div {
+            height: 100px;
+            background-color: lightgray;
+            text-align: center;
+            font-weight: 900;
+            font-size: 90px;
+            color: orange;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="klok"></div>
+    <script src="./showCurrentTime.js">
```

Comment:
I think you renamed your javascirpt file and forgot to update the src here therefore currently code doesn't run 😨